po/ru.po

```diff
@@ -0,0 +1,1271 @@
+
+#: ../data/ui/join_panel.glade:215
+msgid "Min users"
+msgstr "Минимум пользователей"
+
+#: ../src/core/chat_command.c:472
+msgid "Missing argument <addr>"
+msgstr "Пропущен аргумент <addr>"
+
+#: ../src/core/chat_command.c:371
+msgid "Missing argument <server>"
+msgstr "Пропущен аргумент <server>"
+
+#: ../src/lib/command.c:409
+#, c-format
+msgid "Missing argument, expect %1$d, actually %2$d"
+msgstr "Missing argument, expect %1$d, actually %2$d"
+
+#: ../src/sui/sui_config.c:32
+#, c-format
+msgid "Missing field in SuiApplicationConfig: %1$s"
+msgstr "Отсутствует поле в SuiApplicationConfig: %1$s"
+
+#: ../src/core/server_config.c:49
+#, c-format
+msgid "Missing field in server config: %1$s"
+msgstr "Отсутствует поле в конфигурации сервера: %1$s"
+
+#: ../src/core/user_config.c:71
+#, c-format
+msgid "Missing field in user config: %1$s"
+msgstr "Отсутствует поле в конфигурации пользователя: %1$s"
+
+#: ../src/lib/command.c:418
+#, c-format
+msgid "Missing vaule for option %1$s"
+msgstr "Отсутствует значение для опции %1$s"
+
+#: ../data/ui/connect_panel.glade:392
+msgid "NickServ"
+msgstr ""
+
+#: ../data/ui/connect_panel.glade:84
+msgid "Nickname:"
+msgstr "Ник:"
+
+#: ../src/core/app_irc_event.c:1090
+msgid "No acknowledged capability"
+msgstr "Нет подтвержденных возможностей"
+
+#: ../src/core/app_irc_event.c:1047
+msgid "No capability to be requested"
+msgstr ""
+
+#: ../src/core/app_irc_event.c:1080
+msgid "No new capability to be requested"
+msgstr ""
+
+#: ../src/core/chat_command.c:386
+#, c-format
+msgid "No such chat: %1$s"
+msgstr "Нет такого чата: %1$s"
+
+#: ../src/core/chat_command.c:375
+#, c-format
+msgid "No such server: %1$s"
+msgstr "Нет такого сервера: %1$s"
+
+#: ../src/core/app_irc_event.c:1460
+msgid "No topic is set"
+msgstr "Тема не установлена"
+
+#: ../data/ui/connect_panel.glade:352
+msgid "None"
+msgstr ""
+
+#: ../src/lib/path.c:199
+msgid "Not a directory"
+msgstr "Не каталог"
+
+#: ../src/lib/path.c:218
+msgid "Not a regular file"
+msgstr "Не обычный файл"
+
+#: ../src/sui/sui_prefs_dialog.c:165
+msgid "OK"
+msgstr "OK"
+
+#: ../src/sui/sui_app.c:119
+msgid "Open one or more IRC URLs"
+msgstr "Откройте один или несколько URL IRC"
+
+#: ../data/ui/join_panel.glade:100 ../data/ui/connect_panel.glade:186
+msgid "Optional"
+msgstr "Необязательно"
+
+#: ../data/ui/join_panel.glade:88 ../data/ui/connect_panel.glade:368
+#: ../data/ui/connect_panel.glade:409 ../data/ui/connect_panel.glade:464
+msgid "Password:"
+msgstr "Пароль:"
+
+#: ../src/core/app_irc_event.c:246
+msgid "Ping time out"
+msgstr "Истечение времени"
+
+#: ../src/sui/sui_join_panel.c:297
+msgid "Please connect to server before joining any channel"
+msgstr "Подключитесь к серверу, прежде чем присоединяться к каналу"
+
+#: ../src/sui/sui_join_panel.c:377
+msgid "Please connect to server before searching any channel"
+msgstr "Подключитесь к серверу перед поиском канала"
+
+#: ../data/ui/connect_panel.glade:106
+msgid "Quick Mode"
+msgstr ""
```

Comment:
Can this string to be translated?
If you don‘t know the context of this text, free feel to ask me.

This text is title of tab page of server connection panel, this page provides a easy(and quick) way to connect to IRC server.